pkg/apis/v1beta1/ec2nodeclass_status.go

```diff
@@ -38,6 +38,34 @@ type SecurityGroup struct {
 	Name string `json:"name,omitempty"`
 }
 
+// CapacityReservation contains resolved Capacity Reservation selector values utilized for node launch
+type CapacityReservation struct {
+	// ID of the Capacity Reservation
+	// +required
+	ID string `json:"id"`
+	// AvailabilityZone of the Capacity Reservation
+	// +required
+	AvailabilityZone string `json:"availabilityZone"`
+	// Available Instance Count of the Capacity Reservation
+	// +required
+	AvailableInstanceCount int `json:"availableInstanceCount"`
+	// Instance Match Criteria of the Capacity Reservation
+	// +required
+	InstanceMatchCriteria string `json:"instanceMatchCriteria"`
+	// Instance Platform of the Capacity Reservation
+	// +required
+	InstancePlatform string `json:"instancePlatform"`
+	// Instance Type of the Capacity Reservation
+	// +required
+	InstanceType string `json:"instanceType"`
+	// Owner Id of the Capacity Reservation
+	// +required
+	OwnerID string `json:"ownerId"`
```

Comment:
Do we need to add OwnerID here?

Reply:
it helps to identify if the capacity reservation is from own account or a shared one